app/config.js

```diff
@@ -0,0 +1,16 @@
+// Use this file to change prototype configuration.
+
+// Note: prototype config can be overridden using environment variables (eg on heroku)
+
+module.exports = {
+
+  // Service name used in header. Eg: 'Renew your passport'
+  serviceName: 'Set service name in app/config.js',
+
+  // Default port that prototype runs on
+  port: '3000',
+
+  // Enable or disable password protection on production
+  useAuth: 'true'
```

Comment:
Probably better to use a `boolean` here rather than a `string`.

Would then allow you to avoid converting strings to lowercase to check against a string of 'true' in [server.js](https://github.com/alphagov/govuk_prototype_kit/blob/add_config/server.js#L23)